subschemas/substrate-chain/src/typeDefs/chainInfo.ts

```diff
@@ -8,6 +8,11 @@ export default /* GraphQL */ `
     chain: String!
     nodeName: String!
     nodeVersion: String!
+    democracyEnactmentPeriod: String!
+    crowdloanMinContribution: String!
```

Comment:
I have the feeling that we'll have to move these to somewhere else later, but I guess we can keep them here for now. Also some of the constants will not be present in other networks like our parachains (Litmus and Litentry)

Reply:
For now I have made these data optional in case any module is not present in a particular chain.